src/main/java/model/Player.java

```diff
@@ -0,0 +1,45 @@
+package model;
+
+
+import exception.InvalidMoneyAmountException;
+
+public class Player extends Person{
+    private int wallet;
+    private int bet;
+    private int playerNumber;
+
+    public Player(){
+        wallet = 100;
```

Comment:
I would suggest to initialize a new player Number as well, or to add a constructor with the player number:
Player(int playerNumber){ this.wallet = 100; this.playerNumber = playerNumber}

Reply:
Also I would ad this before the variables ( to better understand which variable is ment)